### DISCUSSION TITLE
Multi-game ROM entries

### DISCUSSION THREAD

**Discussion by Alfex3** [RetroAchievements/RAIntegration#792] [Ideas]
This could be used in case someone wants to use a ROM with 2 games which also have separate ROMs. Example: in Super Mario All-Stars + Super Mario World, if SMW is selected, load the SMW set, else load the SMAS set.

**Comment by Sanaki** [comment]
This is already planned for V2, though with loading multiple sets simultaneously. The hash dictates the sets associated with it, and if hash specific logic exists it's provided rather than the standard. I can't see a way implementing this would work sanely in V1.
